requirements-dev.txt

```diff
@@ -10,4 +10,6 @@ torchvision>=0.12.0
 black
 build
 flake8
-psutil
\ No newline at end of file
+psutil
+isort
```

Comment:
Could `isort` be added to .pre-commit-config.yaml?

Reply:
unfortunately, I don't have hands-on experience with pre-commit but chatgpt says, yes:

https://chat.openai.com/share/d6e0ccd3-67d2-4cf3-bb2e-6504a4861586

Reply:
done